src/App.js

```diff
@@ -61,8 +61,40 @@ function App() {
         throw new Error(message);
       }
       const airtableData = await response.json();
+      const sortAscending = (objectA, objectB) => {
```

Comment:
- Why are both sorts called back-to-back?
- These sorts aren't doing anything. Why are they placed in the post function?

Reply:
This one I didn't quite understand where they need to be to work. What would be the best way to correct this?

Reply:
I think i figured some of it out!